src/LightXML.jl

```diff
@@ -2,24 +2,23 @@ __precompile__()
 
 module LightXML
 
-using Compat; import Compat.String
-
-# iteratorsize is new in 0.5, declare it here for older versions. However,
-# we do not actually support calling these, since the traits are not defined
-if VERSION < v"0.5.0-dev+3305"
-    function iteratorsize(v)
-        error("Do not call this on older versions")
-    end
+using Compat
+
+@static if VERSION < v"0.7.0-DEV"
+    const IteratorSize = Base.iteratorsize
 else
-    import Base: iteratorsize, SizeUnknown, IsInfinite, HasLength
+    import Base: IteratorSize
 end
 
+# We do not actually support calling these, since the traits are not defined
+import Base: SizeUnknown, IsInfinite, HasLength
+import Base: start, done, next, show, getindex, show, string, length
 
-if is_windows()
-    const libxml2 = Pkg.dir("WinRPM","deps","usr","$(Sys.ARCH)-w64-mingw32","sys-root","mingw","bin","libxml2-2")
-else
-    const libxml2 = "libxml2"
-end
+const libxml2 =
+    ((VERSION < v"0.7.0-DEV" ? is_windows() : Sys.iswindows())
```

Comment:
use Compat.Sys

Reply:
Thanks - so add `using Compat.Sys`? or `Compat.Sys.iswindows()`?

Reply:
I usually do `import Compat: Sys` and then just use `Sys.iswindows()`